docs/UsersGuide/source/WE2Etests.rst

```diff
@@ -392,56 +343,48 @@ To create a new test category called, e.g. ``new_category``:
 
 
 .. _CreateAltTestNames:
 
 Creating Alternate Names for a Test
------------------------------------
-In order to prevent proliferation of WE2E tests, users might want to use the same
-test for multiple purposes.  For example, consider the test 
-
-  ``grid_RRFS_CONUScompact_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16`` 
-
-in the ``grids_extrn_mdls_suites_community`` category.  This checks for the successful
-completion of the Rocoto workflow running the combination 
-of the ``RRFS_CONUScompact_25km`` grid, the ``FV3GFS`` model for
-ICs and LBCs, and the ``FV3_GFS_v16`` physics suite.  If this test also
-happens to use the inline post capability of the weather model (it currently 
-doesn't; this is only a hypothetical example), then this test can also be used
-to ensure that the inline post feature of the App/weather model (which is
-activated in the App by setting ``WRITE_DOPOST`` to ``"TRUE"``) is working properly.
-Since this test will serve two purposes, it should have two names --- one per purpose.  
-Assume we want to set the second (alternate) name to ``activate_inline_post``.  This
-can be accomplished by creating 
-a symlink named ``config.activate_inline_post.sh``, most appropriately in the ``wflow_features``
-category directory, that points to the configuration file 
+--------------------------------------
+To prevent proliferation of WE2E tests, users might want to use the same test for multiple purposes. For example, consider the test 
+
+   ``grid_RRFS_CONUScompact_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16`` 
+
+in the ``grids_extrn_mdls_suites_community`` category. This checks for the successful
+completion of the Rocoto workflow running the combination of the ``RRFS_CONUScompact_25km`` grid, the ``FV3GFS`` model for ICs and LBCs, and the ``FV3_GFS_v16`` physics suite. If this test also
+happens to use the inline post capability of the weather model (it currently doesn't; this is only a hypothetical example), then this test can also be used to ensure that the inline post feature of the App/Weather Model (which is activated in the App by setting ``WRITE_DOPOST`` to ``"TRUE"``) is working properly.
+Since this test will serve two purposes, it should have two names --- one per purpose. To set the second (alternate) name to ``activate_inline_post``, the user needs to create a symlink named ``config.activate_inline_post.sh`` in the ``wflow_features`` category directory that points to the configuration file 
 
   ``config.grid_RRFS_CONUScompact_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16.sh``
 
-in the ``grids_extrn_mdls_suites_community`` category directory.  
+in the ``grids_extrn_mdls_suites_community`` category directory. 
+
+.. code-block:: console
+
+   ln -sf <path/to/config.grid_RRFS_CONUScompact_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16.sh> <path/to/activate_inline_post>
+
+..
+   COMMENT: Is the above code correct? 
+
```

Comment:
@gsketefian, can you confirm?

Reply:
We'd want this (at least on Hera):
```
ln -fs --relative /path/to/grids_extrn_mdls_suites_community/config.grid_RRFS_CONUScompact_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16.sh /path/to/wflow_features/config.activate_inline_post.sh
```
Note that I made several changes.  The `--relative` flag makes the symlink relative, i.e. within/below the `regional_wflow` directory structure, so that it stays valid when copied to other locations.  (The `--relative` flag may be different on other platforms and/or not exist at all, e.g. on one of the WCOSS machines; so you can put in a disclaimer about it being different on the user's machine.)  I also included the `grids_extrn_mdls_suites_community` and `wflow_features` subdirectories in the paths so it is clearer that these files/symlinks are in different locations.